selftest/tcpconnect/Makefile

```diff
@@ -7,8 +7,11 @@ LIBBPF_OBJ = $(abspath $(OUTPUT)/libbpf.a)
 
 CC = gcc
 CLANG = clang
+GO = go
 
 ARCH = x86
```

Comment:
Is this line needed anymore?

Reply:
It's a convenience variable for folks that have Go installed outside of the PATH, e.g. when testing bleeding-edge Golang binaries with libbpfgo, but can drop it if you prefer :)

Reply:
I was referring to `ARCH = x86` in this case